### ISSUE TITLE
How To Start is incorrect

### ISSUE THREAD

**Issue by ekkis** [tradingview/charting-library-examples#9]
on this page: https://github.com/tradingview/charting-library-examples/tree/master/react-example step 2 of "How To Start" indicate:

> Copy charting_library folder from https://github.com/tradingview/charting_library/ to /public. The earliest supported version of the Charting Library is 1.12.

however, the link provided produces a 404.  where am I supposed to get that folder from?

**Comment by ekkis** [comment]
then it should state so on the page. so that those of use that can access the page don't have to open tickets to figure out why we can't follow the instructions

**Comment by timocov** [comment]
Yes, you are right https://github.com/tradingview/charting-library-examples#what-is-charting-library
